service.go

```diff
@@ -3,8 +3,9 @@ package s3
 import (
 	"context"
 
-	"github.com/aws/aws-sdk-go/aws"
-	"github.com/aws/aws-sdk-go/service/s3"
+	"github.com/aws/aws-sdk-go-v2/aws"
+	"github.com/aws/aws-sdk-go-v2/service/s3"
+	"github.com/aws/aws-sdk-go-v2/service/s3/types"
```

Comment:
I prefer to use `s3types` so that we can avoid conflict with our own `types` package.

Reply:
ping @1445601153

Reply:
OK, i will fixed right now.

Reply:
Now in utils.go, we use the s3 types too,and our type is typ. Maybe i should change it too in case of mistaking one to the other?